docker-compose.yml

```diff
@@ -7,35 +7,13 @@ services:
         dockerfile: Dockerfile-2.7
      volumes:
         - .:/opt/project
-  python33:
-     image: batch-scoring-3.3
-     build:
-        context: .
-        dockerfile: Dockerfile-3.3
-     volumes:
-        - .:/opt/project
-  python34:
-     image: batch-scoring-3.4
-     build:
-        context: .
-        dockerfile: Dockerfile-3.4
-     volumes:
-        - .:/opt/project
```

Comment:
In our README we state that we support all python 3+ versions. I think it is time to update it to 3.5+.
Maybe as a separate PR

Reply:
I agree 👍